### PR TITLE
fix: New form color

### PR DESCRIPTION
**Pull request by sulha199**
## Summary of change
Update the AppInfoForm coloring and font

## Related issues
- https://github.com/supertokens/main-website/issues/897

## Checklist
- [ ] Algolia search needs to be updated? (If there is a new sub docs project, then yes)
- [ ] Sitemap needs to be updated? (If there is a new sub docs project, then yes)
- [x] Checked for broken links? (Run `cd v2 && MODE=production npx docusaurus build`)
- [ ] Changes required to the demo apps corresponding to the docs?

## Remaining TODOs for this PR
NA

### CODE REVIEW

File: v2/src/components/appInfoForm/style.css
Comment (sulha199) [inline v2/src/components/appInfoForm/style.css:167]: font family according to this [chat ](https://supertokens.slack.com/archives/C03HY1RQVB2/p1661862415953629?thread_ts=1661852504.367859&cid=C03HY1RQVB2)

File: v2/src/components/appInfoForm/style.css
Comment (sulha199) [inline v2/src/components/appInfoForm/style.css:167]: when I put a border, it looks too much glowing. So, I remove it.

**Comment by sulha199** [comment]
### Broken link checking: 
image

File: v2/src/components/appInfoForm/style.css
Comment (sulha199) [inline v2/src/components/appInfoForm/style.css:171]: The Design use 0.5px solid border. But browser will treat 0.5px solid border as a 1px solid.
Therefore, it looks too much glowing, unlike the design.

So, I remove it. And it looks more similar to the design.

appInfoform color

Comment (nkshah2) [reply v2/src/components/appInfoForm/style.css:171]: Have you spoken to @mufassirkazi about this?

**Review by nkshah2 — APPROVED** [review]

**Comment by nkshah2** [comment]
Will merge once we get the design approval for this

File: v2/src/components/appInfoForm/style.css
Comment (nkshah2) [reply v2/src/components/appInfoForm/style.css:171]: Feedback from mufassir:
- The border for focus can be 1px
- When focussing on an input field the placeholder value should dissapear

**Review by nkshah2 — CHANGES_REQUESTED** [review]

File: v2/src/components/appInfoForm/style.css
Comment (sulha199) [reply v2/src/components/appInfoForm/style.css:172]: Next time I'll ask Mufassir if there is case like this. Meanwhile, its updated.

File: v2/src/components/appInfoForm/style.css
Comment (sulha199) [inline v2/src/components/appInfoForm/style.css:186]: tried combining these 3 rules into one, but it doesn't work. Probably, the parser breaks because of the multiple `:` on the `:focus::placeholder` if these 3 rules merged into one

Comment (nkshah2) [reply v2/src/components/appInfoForm/style.css:186]: 👍🏼

**Review by nkshah2 — APPROVED** [review]

**Pull request merged**
